### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by garthus1**
A few minor grammatical changes to enhance understanding and optimize translation.

## Author checklist (to be completed by original Author)
- [x ] Is this document a good fit for the Rocky project ?
- [ ] Is this a non-English contribution? 
- [ ] Title and Author MetaTags have been inserted into the document 
- [x ] If applicable, steps and instructions have been tested to work on a real system
- [ x] Did you perform an initial self-review to fix basic typos and grammatical correctness

## Rocky Documentation checklist  (to be completed by Rocky team) 
- [ ] 1st Pass (Check that document is good fit for project and author checklist completed)
- [ ] 2nd Pass (Technical Review - check for technical correctness) 
- [ ] 3rd Pass (Basic Editorial Review)
- [ ] 4th Pass (Detailed Editorial Review and Peer Review)
- [ ] 5th Pass (Include document in TOC)
- [ ] Final pass/approval (Final Review)

### CODE REVIEW

**Comment by garthus1** [comment]
I think it is as good as is possible now. A few minor changes to enhance understanding and optimize translations.

**Comment by EzequielBruni** [comment]
Most of this is alraight by me, but there might be typo here: "If you have not already. Join the conversation on..."

That should not be two separate sentences.

**Comment by garthus1** [comment]
However, perhaps I will change that.

**Comment by garthus1** [comment]
I changed it, yes this may be better.

File: README.md [hunk 1]
```diff
@@ -14,13 +14,13 @@ Where do you come in — both now and in the future? If you have experience with
 
 ## Where to Start
 
-While you can create documentation in any format, the preferred format is Markdown. You can easily create documents in Markdown, plus Markdown is simple to learn. The best way to get started with Markdown is to have a Markdown editor installed and read the tutorial. You can create Markdown files with any text editor, but most Markdown editors allow you to preview what you've input already so they can be very helpful.
+While you can create documentation in any format, the preferred format is Markdown. You can easily create documents in Markdown, plus Markdown is simple to learn. The best way to get started with Markdown is to have a Markdown editor installed and read the tutorial. You can create Markdown files with any text editor, but most Markdown editors allow you to preview what you have inputted already so that they can be beneficial.
```
Comment (SergeCroise) [inline README.md:17]: ... what you have **entered** already ...

**Pull request closed without merging**